casper/src/main/scala/coop/rchain/casper/MultiParentCasperImpl.scala

```diff
@@ -299,7 +299,9 @@ class MultiParentCasperImpl[F[_]
         _ <- EitherT(
               EquivocationDetector.checkNeglectedEquivocationsWithUpdate(b, s.dag, approvedBlock)
             )
-        _      <- EitherT.liftF(Span[F].mark("neglected-equivocation-validated"))
+        _ <- EitherT.liftF(Span[F].mark("neglected-equivocation-validated"))
+        // _      <- EitherT(Validate.phloPrice(b, casperShardConf.minPhloPrice))
+        // _      <- EitherT.liftF(Span[F].mark("phlogiston-price-validated"))
```

Comment:
@stanislavlyalin Sorry Stas. One more minor request. Could you put a comment in codes why these lines are commeted?

Reply:
Maybe it would be even better to have an issue to keep track on it.

Reply:
@zsluedem, I added Tomislav's comment as explanation :grinning:

Reply:
@zsluedem I've added a task to check this validation on our secure list for HF3.

Reply:
Instead of completely comment out this validation let's create _warning_ log message. This way we can see in the logs when we have this situation.

Reply:
@zsluedem, @tgrospic, fixed in [commit](https://github.com/rchain/rchain/pull/3556/commits/e3d85586f57ac3144a8b0e625cff827a52dd83cb)